code/game/machinery/doors/airlock.dm

```diff
@@ -34,6 +34,7 @@
 
 #define AIRLOCK_INTEGRITY_N			 300 // Normal airlock integrity
 #define AIRLOCK_INTEGRITY_MULTIPLIER 1.5 // How much reinforced doors health increases
+#define AIRLOCK_SEAL_MULTIPLIER		 2 // How much extra health airlocks get when braced with a seal
```

Comment:
This needs to be documented as per our [documentation guidelines](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)

Reply:
I think this is resolved now